### PR TITLE
FF Workshop Addition

### PR DESCRIPTION
**Pull request by tuffacton**
adding a full 4 part FF workshop

### CODE REVIEW

File: se-workshop-ff/instruqt-track/03-lab2-release-with-ff/setup-sandbox
Comment (jtitra) [inline se-workshop-ff/instruqt-track/03-lab2-release-with-ff/setup-sandbox:1]: update this to only:

#!/bin/bash

exit 0

File: se-workshop-ff/instruqt-track/04-lab3-targeted-feature-releases/setup-sandbox
Comment (jtitra) [inline se-workshop-ff/instruqt-track/04-lab3-targeted-feature-releases/setup-sandbox:1]: update this to only:

#!/bin/bash

exit 0

File: se-workshop-ff/instruqt-track/05-lab4-governance-and-efficiency/setup-sandbox
Comment (jtitra) [inline se-workshop-ff/instruqt-track/05-lab4-governance-and-efficiency/setup-sandbox:1]: update this to only:

#!/bin/bash

exit 0

**Review by jtitra — CHANGES_REQUESTED** [review]
please update the 'setup-sandbox' for 03-lab2, 04-lab3, and 05-lab4

File: se-workshop-ff/instruqt-track/03-lab2-release-with-ff/setup-sandbox
Comment (tuffacton) [reply se-workshop-ff/instruqt-track/03-lab2-release-with-ff/setup-sandbox:1]: should be resolved

File: se-workshop-ff/instruqt-track/04-lab3-targeted-feature-releases/setup-sandbox
Comment (tuffacton) [reply se-workshop-ff/instruqt-track/04-lab3-targeted-feature-releases/setup-sandbox:1]: should be resolved

File: se-workshop-ff/instruqt-track/05-lab4-governance-and-efficiency/setup-sandbox
Comment (tuffacton) [reply se-workshop-ff/instruqt-track/05-lab4-governance-and-efficiency/setup-sandbox:1]: should be resolved, that can explain why some labs took forever to setup

**Review by tuffacton — COMMENTED** [review]
Should be all good

**Review by jtitra — APPROVED** [review]

**Pull request merged**